icekit/navigation/templates/icekit/navigation/accounts_navigation_item.html

```diff
@@ -0,0 +1,12 @@
+<li class="ik-nav-item ik-nav-item--accounts {% if instance in active_navigation_items %}ik-nav-item--active{% endif %}">
```

Comment:
It seems unwieldy to introduce a new context variable for something that is conceptually part of the `navigation` variable. I'd like to aim for `{% if instance in navigation.active_items %}`.